python/max_flow.py

```diff
@@ -0,0 +1,172 @@
+from math import floor, log2
+from typing import List, Dict
+import mgp
+from itertools import chain
+from mage.max_flow.bfs_weight_min_max import BFS_find_weight_min_max
+
+
+@mgp.read_proc
+def get_flow(
+    context: mgp.ProcCtx,
+    start_v: mgp.Vertex,
+    end_v: mgp.Vertex,
+    edge_property: str = "weight",
+) -> mgp.Record(max_flow=mgp.Number):
+    """
+    Calculates maximum flow of graph from paths found with get_paths()
+
+    :param start_v: source vertex for outgoing flow
+    :param end_v: sink vertex for ingoing flow
+    :param edge_property: property of edge to be used as flow capacity
+
+    return: number value of graph max flow
+
+    The procedure can be invoked in openCypher using the following call:
+    MATCH (source {id: 0}), (sink {id: 5})
+    CALL max_flow.get_flow(source, sink, "weight")
+    YIELD max_flow
+    RETURN max_flow
+    """
+
+    records = get_paths(context, start_v, end_v, edge_property)
```

Comment:
I would rather make another method with different return types, because this one is highly coupled with Memgraph Record

Reply:
Not sure what you mean, my idea is that get_paths is also a query module, used for retrieving paths taken by max flow, this could be useful e.g. for min-cut. get_flow is used to just get the max flow.

Reply:
Memgraph's `mgp.Record` isn't something that should be traversed through, module should be an independent unit, and no one except for Memgraph's internal code should use Record fields in their program